### PR TITLE
Added links to the workshop per #714

### PR DESCRIPTION
**Pull request by thothmoses**
Added links and fixed 1 graphic

References #xxxx

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/pages/grpc-service.adoc [hunk 1]
```diff
@@ -198,9 +198,13 @@ NOTE: The following steps for cloud deployment are optional. If you are only run
 [#kubernetes]
 == Run in Kubernetes
 
-We suggest that you read xref:deployment:index.adoc[] to get an understanding of how deployment with the Akka Cloud Platform works.
+We suggest that you read the xref:deployment:index.adoc[] section to get an understanding of how deployment with the Akka Cloud Platform works.
 
-Before following the steps below, create Kubernetes cluster and install the Akka Operator. Used the instructions below for:
+=== Akka Workshop
+The fifth video of the https://info.lightbend.com/akka-platform-workshop-part-5-on-demand-recording.html[Akka Workshop Series {tab-icon}, window="tab"] walks you through the process of running in Kubernetes on AWS or GCP. It will take you step-by-step to aid you in understanding the process.
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/pages/grpc-service.adoc:204]: shall we include similar link also from https://developer.lightbend.com/docs/akka-platform-guide/deployment/aws-index.html ?

**Review by patriknw — COMMENTED** [review]
good, a few additional suggestions...

**Comment by thothmoses** [comment]
@patriknw added your suggestions

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**